t/CMakeLists.txt

```diff
@@ -9,6 +10,14 @@ execute_process(
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
+# Infer if we are working with a 32/64 bit llvm installation, this is needed
```

Comment:
TODO: Cleanup

This is also wrong on ubuntu. `$CLANG_LIBDIR_SUFFIX` is what we want, *but* thats an internal variable.